liquigraph-core/src/test/java/org/liquigraph/core/io/ChangelogGraphReaderTestSuite.java

```diff
@@ -39,7 +39,7 @@
         SLF4JBridgeHandler.removeHandlersForRootLogger();
     }
 
-    private ChangelogGraphReader reader = new ChangelogGraphReader();
+    private ChangelogGraphReaderImpl reader = new ChangelogGraphReaderImpl();
```

Comment:
per our previous discussion (about implementing clearChecksums as a private method in ChangelogGraphReader, this change won't be necessary

Reply:
done in 167e6deda602faa5e88b77fc4d14e246a8b4fcf5